shared/src/main/java/io/pravega/shared/segment/StreamSegmentNameUtils.java

```diff
@@ -117,6 +122,26 @@ public static boolean isTransactionSegment(String streamSegmentName) {
         return true;
     }
 
+    /**
```

Comment:
Where did all these changes come from?

Reply:
These changes were mainly handy methods to tag operations in tier 2, but they have been rolled back as per your suggestion.